xml/schema/types/warrants.xsd

```diff
@@ -104,6 +104,7 @@
             </xs:sequence>
             <xs:attribute name="systemName" type="systemNameType" use="required"/>
             <xs:attribute name="userName" type="userNameType" />
+            <xs:attribute name="wType" type="xs:string"/>
```

Comment:
Should this be "wtype" or "wType"? This differs from the other schema containing this element.